### DISCUSSION TITLE
Could not find a map of Austrian region "Osttirol"

### DISCUSSION THREAD

**Discussion by sushifan233** [osmandapp/OsmAnd#18220] [Maps Related]
Hello everybody, this is my first input here - hopefully I placed my question on the right place.
First of all many thanks for this great peace of software - I am using OSMAND very often.
I have the following problem:

- I downloaded all regions of Austria, e.g. Burgenland, Kärtnen, .... but not the global Austrian map
- In the settings the entry "Grenzen der heruntergeladenen Karten anzeigen" is set to ON
- On the map now I can see that the region "Osttirol" ist missing. There are no details shown if I zoom into this area

"Osttirol" is in the south of Salzburg and in the west of Kärnten but does not have a direct connection to Tirol. Regarding government Osttirol belongs to Tirol.

Thanks for your support.

**Comment by DmitryAlexei** [comment]
@sushifan233 please add a screenshot, so we could see an error

**Comment by sushifan233** [comment]
Hello, here are two screenshots, one with the map, the other with the used maps.
![Austria-Osttirol](https://github.com/osmandapp/OsmAnd/assets/145968446/e9baec5e-5ff1-403e-aa85-757de66fa29d)
![Austria-UsedMaps](https://github.com/osmandapp/OsmAnd/assets/145968446/08f14081-5d47-4979-9c0b-9e486336f5a6)

**Comment by DmitryAlexei** [comment]
@sushifan233 see https://github.com/osmandapp/OsmAnd/issues/16090
This issue should have been done in 4.4.
What is your app version? Please update and see if your issue persists

**Comment by sushifan233** [comment]
Hi, I updated the app and "Osttirol" is not excluded any more. Many thanks for your support.

**Comment by sushifan233** [comment]
One more question:
There is a global Austrian map which can be downloaded, and there are eight separated maps for the regions Burgenland, Kärnten, ...
Is there any difference between the global map and the eight single maps? Do the eight maps have more details included than the global map - or are they equal?

**Reply by DmitryAlexei** [reply]
Standard maps are equal: maps of the regions and the country as a whole contain the same information. At the same time, in order to avoid possible errors during routing, we recommend to use only one map at a time (region or country)

**Comment by sushifan233** [comment]
Many thanks. I will close this thread as "resolved".
